db/migrate/20220720214646_add_delete_cascade_from_orgs_down.rb

```diff
@@ -0,0 +1,11 @@
+class AddDeleteCascadeFromOrgsDown < ActiveRecord::Migration[7.0]
+  def up
+    remove_foreign_key :schemes, :organisations, column: "owning_organisation_id"
+    remove_foreign_key :schemes, :organisations, column: "managing_organisation_id"
```

Comment:
Why are we removing this foreign key?

Reply:
My migration file was unbalanced, I have added back the foreign key for managing organisations.